src/getExtensionApi.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { Extension, extensions } from "vscode";
+
+export async function getApiExport<T>(extensionId: string): Promise<T | undefined> {
```

Comment:
I think you should go ahead and implement the `getResourceGroupApi` funciton in here since you already have this file.

Just makes the `extension.ts` file cleaner.

https://github.com/microsoft/vscode-azurefunctions/blob/main/src/getExtensionApi.ts#L24-L31